File: devops/openapi/java/src/main/java/com/epam/poc/aicode/devops/openapi/Controller.java
package com.epam.poc.aicode.devops.openapi;


import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.List;


@RestController
@RequestMapping("/api")
public class Controller {

    @GetMapping(value = "/users")
    public List<User> getListOfUsers() {
        return List.of(
            new User.Builder()
                .name("John Doe")
                .email("some-firstE@email.www")
                .password("some-password")
                .address("some-address")
                .phone("some-phone")
                .build(),
            new User.Builder()
                .name("Jane Doe")
                .email("some-secondE@email.www")
                .password("some-password2")
                .address("some-address2")
                .phone("some-phone2")
                .build(),
            new User.Builder()
                .name("Jane Doe")
                .email("some-thirdE@email.www")
                .password("some-password3")
                .address("some-address3")
                .phone("some-phone3")
                .build()
        );
    }
}

File: devops/openapi/C#/VisualStudio/openapi/Controller.cs
using Microsoft.AspNetCore.Mvc;

namespace openapi
{
    [ApiController]
    [Route("api")]
    public class Controller : ControllerBase
    {
        [HttpGet("users")]
        public List<User> GetListOfUsers()
        {
            return new List<User>
            {
                new User
                {
                    Name = "John Doe",
                    Email = "some-firstE@email.www",
                    Password = "some-password",
                    Address = "some-address",
                    Phone = "some-phone"
                },
                new User
                {
                    Name = "Jane Doe",
                    Email = "some-secondE@email.www",
                    Password = "some-password2",
                    Address = "some-address2",
                    Phone = "some-phone2"
                },
                new User
                {
                    Name = "Jane Doe",
                    Email = "some-thirdE@email.www",
                    Password = "some-password3",
                    Address = "some-address3",
                    Phone = "some-phone3"
                }
            };
        }
    }
}
